ucsb-cs156-f24/proj-happycows-f24-11
# Hide previous and next buttons for Profit Table

## Overview

In this PR, when there are 0 or 1 page in the Profit Table, the previous and next buttons are hidden.

## Screenshots (Optional)
![image](https://github.com/user-attachments/assets/b072a5cc-e6a5-4faa-a96f-d80c297b675a)

## Tests

## Dokku link
https://happycows-blobthebuilder.dokku-11.cs.ucsb.edu/

## How to test
Open dokku and create new commons. The next and previous buttons should be hidden.

## Linked Issues

Closes #22
